home-data/src/main/java/com/softserveinc/ita/homeproject/model/Role.java

```diff
@@ -0,0 +1,43 @@
+package com.softserveinc.ita.homeproject.model;
```

Comment:
Why do we name the package - model? Can we name it an entity? Please don't forget also add module affiliation to the package naming.